### ISSUE TITLE
java.lang.NoSuchMethodException: java.lang.ClassLoader.isAncestor warning

### ISSUE THREAD

**Issue by Condor70** [mjiderhamn/classloader-leak-prevention#76]
Using classloader-leak-prevention-servlet3-2.5.0.jar on WebSphere Application Server 9.0.

Apparently, the WebSphere classloader doesn't have an isAncestor method. This is not a problem, but ClassLoaderLeakPreventor.findMethod logs the entire stack trace for NoSuchMethodException as a warning.

This is annoying, because I regularly scan our log files for application exceptions.

Could the logging be reduced to a single line instead of an entire stack trace?

**Comment by mjiderhamn** [comment]
Hi @Condor70 . I'm assuming you are running WebSphere on IBM JDK? What version?

Possibly I should make an exception regarding `isAncestor()` for IBM JDKs.

**Comment by Condor70** [comment]
Correct. I'm using IBM JDK 8:

```
java version "1.8.0_151"
Java(TM) SE Runtime Environment (build 8.0.5.6 - pwa6480sr5fp6-20171124_02(SR5 F
P6))
IBM J9 VM (build 2.9, JRE 1.8.0 Windows 7 amd64-64 Compressed References 2017112
2_371101 (JIT enabled, AOT enabled)
OpenJ9   - 8e3c85d
OMR      - 713f08e
IBM      - c041ee8)
JCL - 20171113_01 based on Oracle jdk8u151-b12
```

**Comment by mjiderhamn** [comment]
A fix for this was included in version 2.6.0 just released to Maven Central.

**Comment by Condor70** [comment]
Thanks. Just tested it and it works perfectly.
